packages/turso-serverless/src/connection.ts

```diff
@@ -223,6 +223,17 @@ export class Connection {
     this.isOpen = false;
     await this.session.close();
   }
+
+  reconnect(): void {
```

Comment:
This should be async method because close() is async, no?

Reply:
ah right, yeah `close()` is async. updated

Reply:
Hm...but you also need to `await this.close()`?

Reply:
yeah makes sense, should i also update files from [tursodatabase/libsql-client-ts#322](https://github.com/tursodatabase/libsql-client-ts/pull/322) to have await on all `.close()` calls?

Reply:
alright